app/lib/irs1040_schedule_eic_pdf.rb

```diff
@@ -0,0 +1,43 @@
+class Irs1040ScheduleEicPdf
+  include PdfHelper
+
+  def source_pdf_name
+    "tax_documents/f1040sei-TY2021"
+  end
+
+  def initialize(submission)
+    @xml_document = SubmissionBuilder::Ty2021::Documents::ScheduleEic.new(submission).document
+    @intake = submission.intake
+  end
+
+  def hash_for_pdf
+    answers = {
+      FullPrimaryName: @intake.primary_full_name,
+      PrimarySSN: @intake.primary_ssn
+    }
+    dependent_nodes = @xml_document.search("QualifyingChildInformation")
+    answers.merge!(dependents_info(dependent_nodes[0..]))
```

Comment:
does calling with the range at the end transform the data somehow? Wondering what the distinction between `dependent_nodes` and `dependent_nodes[0..]`? Are they different data types?